src/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -105,213 +86,216 @@ private static void WriteTestDllAndName(TextWriter writer, string dll, string da
         /// <param name="level">Specifies the level of granuality to use for the output: either namepspace, class or function</param>
         /// <param name="writer">The StreamWriter which will be passed to WriteTestDllAndName() </param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", MessageId = "System.Reflection.Assembly.LoadFrom")]
-        public static void GetTests(IList<string> dlls, Level level, TextWriter writer)
+        public static void GetTests(IList<string> dlls, Level level)
```

Comment:
[Should] Maybe it would be better to have this function return a list of strings `"<DLL path> | <test name>"` and then let Main() either write them to stdout or to a file.  Then the name GetTests() would make sense.

Reply:
This has been changed in the next version.